launcher/ui/widgets/ThemeCustomizationWidget.ui

```diff
@@ -40,40 +40,80 @@
     </widget>
    </item>
    <item row="0" column="1">
-    <widget class="QComboBox" name="iconsComboBox">
-     <property name="sizePolicy">
-      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
-       <horstretch>0</horstretch>
-       <verstretch>0</verstretch>
-      </sizepolicy>
-     </property>
-     <property name="focusPolicy">
-      <enum>Qt::StrongFocus</enum>
-     </property>
-    </widget>
+    <layout class="QHBoxLayout" name="iconsLayout">
+     <item>
+      <widget class="QComboBox" name="iconsComboBox">
+       <property name="sizePolicy">
+        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
+         <horstretch>0</horstretch>
+         <verstretch>0</verstretch>
+        </sizepolicy>
+       </property>
+       <property name="focusPolicy">
+        <enum>Qt::StrongFocus</enum>
+       </property>
+      </widget>
+     </item>
+     <item>
+      <widget class="QPushButton" name="iconsFolder">
+       <property name="toolTip">
+        <string>View icon themes folder.</string>
+       </property>
+       <property name="text">
+        <string/>
+       </property>
+       <property name="icon">
+        <iconset theme="viewfolder"/>
+       </property>
+       <property name="flat">
```

Comment:
make not flat

Reply:
i think there was a height difference with the combobox so i just hid it

maybe best to use something other than a button for the info icon...- or is that not possible?

Reply:
seems you can use a label. still needs C++.